src/components/molecules/MageInformation/index.tsx

```diff
@@ -25,11 +25,19 @@ type Props = {
   theme: any
 }
 
+function formatCharges(mage: Mage): string {
+  var res = mage.numberOfCharges.toString();
+  if (mage.numberOfOvercharges) {
+    res += ` + ${mage.numberOfOvercharges}`;
+  }
+  return res;
```

Comment:
Maybe we could also make this immutable:

```ts
function formatCharges(numberOfCharges: number, numberOfOvercharges: number): string {
    return `${numberOfCharges}${numberOfOvercharges ? ` + ${numberOfOvercharges}` : '' }`
}
```

Reply:
note: I also changed the functions arguments, so that it does only take what it actually needs. I don't think we should hand over the whole mage object here :)

Reply:
Done (with the mod that numberOfCharges is number | 'Custom')